### PR TITLE
Remove getter and access tool.issue

### PR DESCRIPTION
**Pull request by ConorDavenport**
To get the issue detected by doctor, access the issue member of doctor directly instead of using the getIssue() function

See https://github.com/nearform/node-clinic-doctor/pull/323 for Doctor issue member PR

### CODE REVIEW

**Comment by goto-bus-stop** [comment]
It won't throw an error now if `.issue` doesn't exist, but `tool.issue` might be undefined. The `iss !== 'none'` check needs to take that into account.

We would typically update the `@clinic/doctor` version requirement before doing a release so there should not be a case where an older doctor version is used.

File: bin.js [hunk 1]
```diff
@@ -611,24 +611,24 @@ async function runTool (args, Tool, version, uiOptions) {
...
-        console.log(`clinic bubbleprof --autocannon [ / ] -- node ${proc}`)
-      } else if (iss === 'data') {
-        console.log('Try running the benchmark for a longer time.')
-      }
-    } catch (err) {
-      // The version of Doctor used does not have a getIssue() function
+  if (Tool.name === 'ClinicDoctor' && !args['collect-only'] && !args['visualize-only']) {
+    const iss = tool.issue
+
+    const argsArr = args['--']
```
Comment (goto-bus-stop) [inline bin.js:617]: The `.shift()` call below will mutate the original `args['--']` array, which maaaay not be a problem in this case but would definitely be unexpected if some other place in the code needs it in the future.

You can create a copy using the `.slice()` method, but even better, here you can do:

```js
args['--'].slice(1)
```
to get a new array that has all the elements from the `args['--']` array, _skipping_ the 1st element.

Comment (ConorDavenport) [reply bin.js:617]: Thanks! That's sorted now

**Comment by goto-bus-stop** [comment]
Reverted the previous PR in `master` and created a new PR #245 in case this work can be picked up in the future.

**Pull request closed without merging**